TypeCobol.Analysis.Test/PerformanceWithCfg.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TypeCobol.Compiler;
+using TypeCobol.Compiler.Diagnostics;
+using TypeCobol.Compiler.Directives;
+using TypeCobol.Compiler.Text;
+using TypeCobol.Test.Compiler.Parser;
+using TypeCobol.Test.Utils;
+
+namespace TypeCobol.Analysis.Test
+{
+    [TestClass]
+    [Ignore]
+    public class PerformanceWithCfg : TypeCobol.Test.Parser.Performance.Performance
+    {
+        /// <summary>
+        /// Method for parsinga document.
+        /// </summary>
+        /// <param name="fullPath"></param>
+        /// <param name="options"></param>
+        /// <param name="format"></param>
+        /// <param name="copiesFolder"></param>
+        /// <returns></returns>
+        protected override TypeCobol.Parser parseDocument(string fullPath, TypeCobolOptions options, TypeCobol.Compiler.DocumentFormat format, string[] copiesFolder)
+        {
+            //Add analyzers
+            var analyzerProvider = new CompositeAnalyzerProvider();
+            //CFG/DFA
+            const string cfgDfaId = "cfg-dfa";
+            analyzerProvider.AddActivator((o, t) => CfgDfaAnalyzerFactory.CreateCfgAnalyzer(cfgDfaId, CfgBuildingMode.Standard));
+
+            var document = new TypeCobol.Parser();
+            document.Init(fullPath, options, format, copiesFolder, analyzerProvider);
+            document.Parse(fullPath);
+            return document;
+        }
+
+        [TestMethod]
+        [TestCategory("Performance")]
+        [TestProperty("Time", "fast")]
+        //[Ignore]
+        public new void AntlrPerformanceProfiler()
```

Comment:
You can remove AntlrPerformanceProfiler test, because CodeElement/Antlr parsing is not affected by CFG.

Reply:
done in af9e8cb